### ISSUE TITLE
Avatar and Chip - Provide alternative text for images - (2036569783)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5564]
### Module: 
 07b Avatar and Chip 

### Violation: 
 Provide alternative text for images 

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036569783 

### Severity: 
 10 

### Description: 
 [Issue]
There are meaningful images without a textual equivalent. Examples include:
- Test 2. Medium Avatar Thumbnail
- Test 4. Small Avatar Thumbnail

[User Impact]
Screen reader users will be unable to determine what these images represent.

[Code Reference] 
 ```
N/A
 ``` 

### Note: 
 [Suggestion]
Provide alternative text for images. Meaningful images must have a concise but descriptive textual equivalent. To add a textual equivalent to an ``` <img> ``` element, set its alt attribute to a descriptive value. To add a textual equivalent to a ``` <svg> ``` element, add an aria-label attribute set to the desired equivalent.

[Compliant Code Example] 
 ```
N/A
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
For reference, the [testing page](https://geospatialem.github.io/calcite-samples/avatar.html) for tests 2 and 4.

**Comment by geospatialem** [comment]
For a solution, we should pursue requiring an existing prop, such as `username` (or similar) to assign as an `aria-label` to provide context to assistive technologies.

Alternatively we could have a default `aria-label` assigned, and when the user provides a value to the `username` (or similar prop), assign the value to the `aria-label`.

**Comment by geospatialem** [comment]
Verified in `1.4.0-next.9` with Windows JAWS and NVD. Also includes the fix for #5563 where the role is set to figure and provided an `aria-label`.
